ROMFS/px4fmu_common/init.d/rc.interface

```diff
@@ -128,16 +121,24 @@ then
 	then
 		if fmu mode_pwm
 		then
-			if mixer load $OUTPUT_AUX_DEV $MIXER_AUX_FILE
+			if [ -f $OUTPUT_AUX_DEV ]
```

Comment:
When you invest the effort to test, could you just push a commit onto the branch? Its not that I "own" the pull request and its much better to have what you tested in a commit, rather than me interpreting what you mean verbally and doing an open loop change (or needing to wait for another test (which is tedious for you as well) from your side.